ssz-rs/src/ser.rs

```diff
@@ -66,6 +71,13 @@ pub fn serialize_composite_from_components(
     if total_size as u64 >= MAXIMUM_LENGTH {
         return Err(SerializeError::MaximumEncodedLengthExceeded(total_size))
     }
+    // TODO: should this be == instead?
+    if fixed.len() > variable_lengths.len() {
```

Comment:
as currently written, `fixed` and `variable_lengths` are _always_ equal in size, it is an error via the caller if this is not the case... could do w/ a `debug_assert!` possibly although after looking at this code I may try to simplify some stuff...

Reply:
see https://github.com/ralexstokes/ssz-rs/pull/65

Reply:
Switched to `debug_assert_eq!` here, so it'll be easier to merge with #65